### PR TITLE
Deposit Transaction Container

### PR DESCRIPTION
**Pull request by aostrun**
Resolves: #68

### CODE REVIEW

File: src/renderer/components/CopyField/CopyField.tsx [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
     return(
-        <div className="copy-field">
-            {props.value}
-            <CopyButton onClick={props.onCopy}/>
-        </div>
+        <>
+            <h3 className="copy-field-label">{props.label}</h3>
+            <div className="copy-field">
+                <CopyButton onClick={copyToClipboard}/>
+                <div className="copy-field-body">{props.value}</div>
```
[the remark this answers is not in the document]
Comment (aostrun) [reply src/renderer/components/CopyField/CopyField.tsx]: Fixed, I missed the `MnemonicCopyField` component which was unnecessary copy of the `CopyField`. Now it should all work fine.

**Comment by aostrun** [comment]
> tx conainer needs to be larger and probably raised a bit so it doesn't touch bottom of screen. try to scale down some title margings as well

We should refactor the `Modal` component as it was not designed to be responsive (single fixed width does not work well with responsive content). So this is not completely the fault of the `DepositTxContainer` but rather a parent `Modal` component. I'll work on that.

**Comment by mpetrunic** [comment]
> We should refactor the `Modal` component as it was not designed to be responsive (single fixed width does not work well with responsive content). So this is not completely the fault of the `DepositTxContainer` but rather a parent `Modal` component. I'll work on that.

I don't think wider modal will help you, vertical size is issue (this container has too much stuff). If you put stuff side by side it will probably be ugly though

**Comment by mpetrunic** [comment]
@aostrun When I checkout branch and run storybook there is bunchof issues with this PR:
- dropdown doesn't display options
- copy button in CopyField and MnemonicField changes position when hovered over

**Comment by aostrun** [comment]
> * dropdown doesn't display options

Dropdown list is updated to render relative to the parent (dropdown input) as previously it was rendered fixed to the screen which caused various problems (scrolling for example would leave dropdown list fixed on the screen in the same position while parent was being scrolled off-screen). Also, the dropdown list does not interfere with the rest of the DOM so it does not reserve space on the page, that's why we had problems with storybook as dropdown was rendering offscreen, now it should work just fine.

> * copy button in CopyField and MnemonicField changes position when hovered over

This problem is described in the issue #83, tooltips should not be rendered as a part of the image so that error will be fixed by implementing tooltip component. I'm planning to implement it ASAP.

**Comment by mpetrunic** [comment]
@aostrun awesome, let's fix this and resolve merge conflict so we can merge this container

**Pull request merged**